src/viewer/NoViewer/CallToAction.jsx

```diff
@@ -8,13 +8,15 @@ import {
   NOVIEWER_DESKTOP_CTA
 } from 'components/pushClient'
 import styles from '../styles'
+import Config from '../../drive/config/config.json'
 
 export default class CallToAction extends Component {
   state = {
     mustShow: false
   }
 
   async componentDidMount() {
+    if (Config.banner.isActivated !== true) return
```

Comment:
is this `banner` or `desktopPromo`?

Reply:
renamed to promoteDesktop : true / falme